docs/heritage.rst

```diff
@@ -0,0 +1,45 @@
+*************
+Code Heritage
+*************
+
+``pfsspy``
+==========
+
+As of initial release, the main component of ``sunkit-magex`` (`sunkit_magex.pfss`) is 
```

Comment:
I am going to be annoying, but we can we get one sentence per line?

Reply:
yep will fix!